### DISCUSSION TITLE
How to change screen brightness on Ubuntu 22.10

### DISCUSSION THREAD

**Discussion by oo92** [regolith-linux/regolith-desktop#784] [Q&A] [answered]
Hi.

I don't know how to adjust the screen brightness in Regolith on Ubuntu 22.10? I couldn't find anything on this online..

**Comment by kgilmer** [comment, marked as answer]
Hi @oo92 , I just use the keys on my laptop that have labels for brightness.  You could also adjust via commandline: http://sandipbgt.com/2015/10/01/control-screen-brightness-from-commandline-in-ubuntu/
